TacticianBundle.php

```diff
@@ -11,8 +10,10 @@ class TacticianBundle extends Bundle
     public function build(ContainerBuilder $container)
     {
         parent::build($container);
-        $container->addCompilerPass(new DoctrineMiddlewarePass());
-        $container->addCompilerPass(new CommandHandlerPass());
+        $container->addCompilerPass(new Compiler\DoctrineMiddlewarePass());
```

Comment:
The security- and validator middleware were in services.yml and now they have compiler passes. I was wondering why you decided to change this? 

This is a question without an opinion (at this time_, I'm just curious toward the motivation.